docs/guides/recovery/user-recovery.primary.in.rst

```diff
@@ -0,0 +1,67 @@
+Troubleshooting advice for RSP login issues.
+
+
+Problems with a new account
+===========================
+
+Attempts to login will not be successful until your account petition is approved by Rubin staff
+(i.e., step 8 of the process for :doc:`../getting-started/get-an-account`).
+This can take up to a week.
+Your patience is appreciated.
+
+Sometimes this approval process requires that Rubin staff verify that you have Rubin data rights.
+Please check your email's spam folder for a data rights verification
+request from `Heather Shaughnessy <mailto:sheather@slac.stanford.edu>`__.
+
+If you are at an institution outside the US or Chile, and you are a member of an international
+in-kind contribution team, but your name does not yet appear on the
+`list of international data rights holders <https://www.lsst.org/scientists/international-drh-list>`__,
+contact your program manager and Heather Shaughnessy to request to be added to the list.
+
+
+Being redirected to the new account setup process
+=================================================
+
+In cases where attempts to login are redirected into the process for setting up a new account,
+the issue is usually that the wrong identity provider was chosen at login.
+
+Try your other identities (e.g., GitHub, ORCID).
```

Comment:
So the problem is that when users randomly try identities they may or may not have used, if they try an identity that they have not used, they will be presented by the sign-up flow. They they are in a hurry and go "ok whatever" and this is how we end up with users with multiple accounts which is (1) more work for Heather and (2) more work for us if they want the files in their original account back. 

Frankly I think it would be less work all around to advise users to email Heather / account-help and ask what identity they used last time.

Reply:
Added in a "stop and ask for help to find out which identity your account is associated with"